astronomer/providers/snowflake/sensors/snowflake.py

```diff
@@ -90,20 +90,13 @@ def execute(self, context: Context) -> None:
                 method_name=self.execute_complete.__name__,
             )
 
-    def execute_complete(
-        self,
-        context: Context,
-        event: dict[str, str | list[str]] | None = None,
-    ) -> Any:
+    def execute_complete(self, context: Context, event: dict[str, Any] | None = None) -> Any:
         """
         Callback for when the trigger fires - returns immediately.
         Relies on trigger to throw an exception, otherwise it assumes execution was
         successful.
         """
         if event:
             if "status" in event and event["status"] == "error":
-                raise AirflowException(event["message"])
-
+                handle_error(self.soft_fail, event["message"])
             self.log.info(event["message"])
-        else:
-            self.log.info("%s completed successfully.", self.task_id)
```

Comment:
By removing these lines, our docstring seems to be inaccurate. Should we keep these lines? Or update docstring

Reply:
This is just a log statement and will no event on end task status but I have restored it.